lib/aida/bot.ex

```diff
@@ -262,10 +278,26 @@ defmodule Aida.Bot do
     Crypto.encrypt(data, public_keys)
   end
 
+  def send_message(%{session: %{do_not_disturb: do_not_disturb}} = message) when do_not_disturb do
+    log_not_sent_do_not_disturb(message)
+  end
+
   def send_message(%{session: session} = message) do
     log_outgoing(message)
 
     channel = ChannelProvider.find_channel(session)
     channel |> Channel.send_message(message.reply, session)
   end
+
+  def set_session_do_not_disturb(message) do
+    %{message | session: %{message.session | do_not_disturb: Message.is_unsubscribe_keyword(message)}}
+  end
+
+  def clean_reply_if_do_not_disturb(%{session: %{do_not_disturb: do_not_disturb}} = message) when do_not_disturb do
+    %{message | reply: []}
```

Comment:
This behavior exposes the fact that we need some sort of "ack" message to indicate that the unsubscribe request was received and that the bot won't send any further messages. I'll talk with @jkicillof about this. Maybe sending a 👌?

Reply:
Reply with the unsubscribe `acknowledge_message`, but from the handle function, allowing the front_desk to take control of the reply